probs/probs.go

```diff
@@ -16,6 +16,7 @@ const (
 	RateLimitedProblem    = ProblemType("urn:acme:error:rateLimited")
 	BadNonceProblem       = ProblemType("urn:acme:error:badNonce")
 	InvalidEmailProblem   = ProblemType("urn:acme:error:invalidEmail")
+	CAAProblem            = ProblemType("urn:acme:error:caa")
```

Comment:
Can you also add a convenience function for this `ProblemType` ala `probs.ConnectionFailure`?